docs/ref/models/expressions.txt

```diff
@@ -163,6 +163,26 @@ the field value of each one, and saving each one back to the database::
 * getting the database, rather than Python, to do work
 * reducing the number of queries some operations require
 
+.. _string-expressions-using-f:
+
+Using string-expressions in ``F()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 4.1
+
+For fields that are string-expressions (or
```

Comment:
Not sure about this. Is a text field a string expression?

Reply:
We can be explicit, for example:
```
For string-based fields, text-based fields, and :class:`~django.contrib.postgres.fields.ArrayField`, you can ..
```